pkg_ci.Rmd

```diff
@@ -60,6 +60,8 @@ Details of how to run tests/checks using different versions of R locally can be
 
 You can fine tune the deployment of tests with each versions by using a testing matrix. See this [more complex example](https://github.com/r-hub/rhub/blob/master/.travis.yml#L5) of a Travis configuration where pkgdown documentation is built during the release R version testing only.
 
+For a package depending on Bioconductor packages, or a Bioconductor package, [Travis options allow to test on either Bioconductor release or devel (or both)](https://docs.travis-ci.com/user/languages/r/#bioconductor).
```

Comment:
@annakrystalli this paragraph is lacking a sentence about the actual desired build matrix (I could add a link to an existing relevant config file? which one?)